tophatmonocle/hivemind-service
# Cross referencer

## Description / JIRA Links
Item         | Description
------------ | -------------
[SERVINFRA-1236](https://tophat.atlassian.net/browse/SERVINFRA-1236) | Title


## DevQA

### DevQA Prep

### DevQA Steps
1) Change the start and end dates in `sample_work_history_first_person` and `sample_work_history_second_person`, in time from epoch. To get time from epoch, go to https://www.epochconverter.com/

2) For `present`, use max int value `9223372036854775807`, otherwise default all dates to the first of the months

3) Run and go to http://127.0.0.1:8888/linkedin_profile_data/match/ and in the response, in `overlap_dates` you should see a list of dates and companies where both profiles are overlapping

## Information checklist

### Tests
- Insufficient tests.

### Code Review
- This PR addresses comments from a previous PR (insert link to previous PR)
